viite-UI/src/view/TileMapSelector.js

```diff
@@ -42,13 +42,9 @@
     });
     $('#roadsVisibleCheckbox').change(function () {
       applicationModel.toggleRoadVisibility();
-      switch(applicationModel.getSelectedLayer()) {
```

Comment:
If we remove this switch both main layers are set visible or not visible when the checkbox is clicked. Is this what we want? Do we want to show LinkPropertyLayer also in project mode and vice versa?